airframe-sql/src/test/scala/wvlet/airframe/sql/analyzer/TypeResolverTest.scala

```diff
@@ -259,30 +259,30 @@ class TypeResolverTest extends AirSpec {
     test("parse WITH statement") {
       val p = analyze("with q1 as (select id from A) select id from q1")
       p.outputAttributes.toList shouldBe List(
-        ResolvedAttribute("id", DataType.LongType, None, Seq(SourceColumn(tableA, a1)), None)
+        ResolvedAttribute("id", DataType.LongType, Some("q1"), Seq(SourceColumn(tableA, a1)), None)
```

Comment:
Should the qualifier be removed at resolving time in this case?

Reply:
Generally speaking, the qualifier should be preserved for handling such cases:
```with q1 as (select id), q2 as (select id)` select q1.id, q2.id from q1, q2```

Reply:
Got it!